frontend/src/components/molecules/FacetedSearchGroup/FacetedSearchGroup.js

```diff
@@ -127,12 +122,33 @@ class FacetedSearchGroup extends Component {
             </Panel>
             <Panel
               key="9"
-              id="rangepicker-panel"
+              id="facet-dates"
               header="Dates"
               className="fs-group-panel"
             >
               <RangePicker picker="month" size="small" />
             </Panel>
+            <Panel
```

Comment:
Just a suggestion but I see a lot of places where very similar elements are repeated. Perhaps a cleaner way to handle this would be to generate these with a map? https://www.carlrippon.com/repeat-element-n-times-in-jsx/

Reply:
Done (there was no need to use a map thought)

Reply:
This will have to be fixed later